modules/eco-node-health-check-operator-creating-node-health-check.adoc

```diff
@@ -0,0 +1,23 @@
+// Module included in the following assemblies:
+//
+//nodes/nodes/eco-node-health-check-operator.adoc
+
+:_content-type: PROCEDURE
+[id="eco-node-health-check-operator-creating-node-health-check_{context}"]
+= Creating a node health check
+Using the web console, you can create a node health check to identify unhealthy nodes and specify the remediation type and strategy to fix them.
+
+.Procedure
+
+. From the *Administrator* perspective of the {product-title} web console, click *Compute* -> *NodeHealthChecks* -> *CreateNodeHealthCheck*.
+. Specify whether to configure the node health check using the *Form view* (the default setting) or the *YAML view*.
+. Enter a *Name* for the node health check. The name must consist of lower case, alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character.
+. Specify the *Remediator* type, *Self-node remediation* or *Other*, and the *Remediation strategy*. The Self-node remediation options are part of the the Self Node Remediation operator that is installed with the Node Health Check operator. Selecting *Other* requires an *API version*, *Kind*, *Name*, and *Namespace* to be entered from an existing resource of a remediator.
```

Comment:
The remediation strategy applies only for SNR.

Reply:
I think an explanation of the remediation strategies options are would be helpful. What Node deletion does and what Resource deletion does

Reply:
Might be worth mentioning that in case of "Other" selection, if the user wants to include also a fieldPath, resourceVersion or uid of the resource, he can do so in the yaml view

Reply:
> It should be Self node remediation and not Self-node remediation (remove the dash)

Thanks. The dash is now removed.

Reply:
> The remediation strategy applies only for SNR.

Thanks. I have added a sentence to clarify this.

Reply:
> maybe include this link when mentioning the Self node remediation operator? https://www.medik8s.io/remediation/self-node-remediation/self-node-remediation/

Thanks. But we do not include links to upstream documentation.

Reply:
> Might be worth mentioning that in case of "Other" selection, if the user wants to include also a fieldPath, resourceVersion or uid of the resource, he can do so in the yaml view

Thanks. The note on the UI already covers this. 
'Note: Some fields may not be represented in this form view. Please select 'YAML view' for full control.'